scripts/run-service.sh

```diff
@@ -9,6 +9,15 @@ python3 manage.py collectstatic \
     --ignore 'tmp*' \
     --noinput
 
+# Defaults to stdout for local development
+ACCESS_LOG_FILE="-"
+
+if [ "$DJANGO_SETTINGS_MODULE" = "config.settings_stage" ]; then
+    ACCESS_LOG_FILE="/var/log/stage-access.log"
+elif [ "$DJANGO_SETTINGS_MODULE" = "config.settings_prod" ]; then
+    ACCESS_LOG_FILE="/var/log/prod-access.log"
```

Comment:
A stupid question :smile: Does everything what goes to `log` directory goes automatically to Splunk or do the files need to be named in some specific way?

Reply:
Thank you for the explanation!